CONTRIBUTING.md

```diff
@@ -25,10 +25,10 @@ brew tap bow-swift/nef
 brew install nef
 ```
 
-- Clone the repository for Bow.
-- Go to the folder where you have cloned Bow and open `contents/Documentation`.
-- Run `pod install` to set up the project.
-- Open `Documentation.xcworkspace`.
+- Clone the repository for Bow
+- Go to the folder where you have cloned Bow
+- Run `nefc install Documentation.app` to set up the project with its dependencies
```

Comment:
I'd try to rely only on the main `nef` entry point and replace this by `nef compile Documentation.app`, since that also installs the dependencies and does an initial sanity check for the compilation of the documentation.
```suggestion
- Run `nef compile Documentation.app` to set up the project with its dependencies
```